ios/RNAccountKitViewController.m

```diff
@@ -58,7 +58,12 @@ - (void)loginWithPhone: (RCTPromiseResolveBlock)resolve
         UIViewController<AKFViewController> *viewController = [_accountKit viewControllerForPhoneLoginWithPhoneNumber:prefillPhoneNumber state:inputState];
         [self _prepareLoginViewController:viewController];
         UIViewController *rootViewController = [UIApplication sharedApplication].delegate.window.rootViewController;
-        [rootViewController showViewController:viewController sender:nil];
+        
+        if([_viewControllerMode isEqualToString:@"present"]) {
+          [rootViewController showDetailViewController:viewController sender:nil];
```

Comment:
Shouldn't be this line as it was before? is there a reason to change that?

```objective-c
[rootViewController presentViewController:viewController animated:YES completion:NULL];
```

Reply:
Yah, I reverted it to the old way.